docker-compose.yaml

```diff
@@ -103,6 +103,7 @@ services:
       - VITE_STRICT=${FRONTEND_STRICT}
     ports:
       - 3000:3000
+      - 6006:6006
```

Comment:
This is needed to run Storybook via Docker, right? At least I couldn't get it to work without.

Reply:
Yes, that's correct. To expose Storybook's port